### DISCUSSION TITLE
Thank You

### DISCUSSION THREAD

**Discussion by interconnectedMe** [morrownr/8814au#123] [General]
I don't know who you are, but I just installed Ubuntu, dual boot, separate drives alongside Windows 10 after failing miserably with Linux Mint for some reason.  I don't know how long I've spent trawling the interwebs trying to get wifi working, but this set of very comprehensive instructions did it for me.

So, thank you very much morrownr, and the community that have contributed to this, wherever you may be.

**Comment by morrownr** [comment]
Hi @interconnectedMe 

Now you know who I am. Glad the driver worked for you. Since folks tend to search the interwebs and come directly to the repo, they often miss the site Main Menu:

https://github.com/morrownr/USB-WiFi

The first 2 menu items have a lot of good information. The rest of the menu items are rather technical and you know if you need them. Issues and Discussions are used to ask about problems, post news and talk about things.

Regards,

@morrownr
